ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -956,6 +971,16 @@ - (BOOL)shouldPresentStep:(ORKStep *)step {
     return shouldPresent;
 }
 
+- (BOOL)shouldReviewStep:(ORKStep *)step {
+    BOOL shouldReview = ([step isKindOfClass:[ORKQuestionStep class]] || [step isKindOfClass:[ORKFormItem class]]);
+    
+    if (shouldReview && [self.delegate respondsToSelector:@selector(taskViewController:shouldReviewStep:)]) {
+        shouldReview = [self.delegate taskViewController:self shouldReviewStep:step];
```

Comment:
I am not total convinced to add this delegate method.
Can you give a use case when a developer want to exclude a step from review? 
If this delegate has a good use, then it would be nice to include the actual reviewStep as a parameter (multiple review step in a task?)